packages/enzyme/src/Debug.js

```diff
@@ -32,7 +32,11 @@ function propString(prop) {
     case 'boolean':
       return `{${prop}}`;
     case 'object':
-      return '{{...}}';
+      if (prop instanceof String) {
+        return `"${prop.toString()}"`;
```

Comment:
`String(x)` is always strictly better than `x.toString()`, and inside a template literal, both are unnecessary - this can just be ``` `"${prop}"` ```